### DISCUSSION TITLE
Numbers are formatted with too much zero's and no thousand separator

### DISCUSSION THREAD

**Discussion by SlimPens** [apexcharts/apexcharts.js#4025] [Q&A]
Hi All,

The Apex chart which I have adds 8 zero's to each number, no thousand separator and no EURO prefix.

The output is currently: 600.000,00001234

This should be : € 600.000

how can i add this to the JS code:

This is my code:

`        var formatter = new Intl.NumberFormat('de-DE', {
         style: 'currency',
         currency: 'EUR',
          minimumFractionDigits: '0',
          maximumFractionDigits: '0',
});
/`

**Comment by it-xtech-dev** [comment]
Well,

evaluatating:
`new Intl.NumberFormat('de-DE', { style: 'currency', currency: 'EUR' }).format(600.000,00001234)
`
results with:
'600,00 €'

If this kind of formatting is fine for you, try (xaxis) yaxis formatter:
https://apexcharts.com/docs/options/yaxis/

```
// before options initialization:
// NOTE: you can customise format options as you like
const euro = Intl.NumberFormat('de-DE', { style: 'currency', currency: 'EUR' })

// in chart options
{
  xaxis: {
    labels: {
      formatter (val) {
         return euro.format(val)
      }
    }
  }
}

```

IT.man
